cmd/immuclient/cli/flags.go

```diff
@@ -71,17 +70,21 @@ func parseflags() error {
 	if err := viper.BindPFlag("value-only", pflag.Lookup("value-only")); err != nil {
 		return err
 	}
+	if err := viper.BindPFlag("config", pflag.Lookup("config")); err != nil {
+		return err
+	}
 	pflag.Parse()
 	o.InitConfig(o.CfgFn)
-	viper.SetDefault("immudb-port", gw.DefaultOptions().ImmudbPort)
-	viper.SetDefault("immudb-address", gw.DefaultOptions().ImmudbAddress)
-	viper.SetDefault("auth", client.DefaultOptions().Auth)
```

Comment:
@padurean it's correct to remove auth flag?

Reply:
@mmeloni I know this was @volknanebo s dream :) since the first time he saw it and if i remember correctly it was agreed we should remove it. In `immuadmin` it is removed already and it works like this: if the token file is present and can be read, the client will send the token (i.e. the token file contents). So no flag needed. `immuclient` can do the same, so if it is tested and it works in both cases (with and without auth disable in `immudb` server), then it's fine.

Reply:
@padurean :)
Yes , auth flag removed.